UnitTests/UnitTests.UWP/Extensions/Test_VisualTreeExtensions.cs

```diff
@@ -119,6 +157,42 @@ public async Task Test_VisualTree_FindDescendant_Exists()
             });
         }
 
+        [TestCategory("VisualTree")]
+        [TestMethod]
+        [DataRow(SearchType.DepthFirst)]
+        [DataRow(SearchType.BreadthFirst)]
+        public async Task Test_VisualTree_FindDescendant_Exists(SearchType searchType)
+        {
+            await App.DispatcherQueue.EnqueueAsync(async () =>
+            {
+                var treeRoot = XamlReader.Load(@"<Page
+    xmlns=""http://schemas.microsoft.com/winfx/2006/xaml/presentation""
+    xmlns:x=""http://schemas.microsoft.com/winfx/2006/xaml""> <!-- Starting Point -->
+    <Grid>
+        <Grid>
+            <Border/>
+            <StackPanel>
+                <TextBox/>
+                <TextBlock/> <!-- Target -->
+            </StackPanel>
+        </Grid>
+    </Grid>
+</Page>") as Page;
+
+                // Test Setup
+                Assert.IsNotNull(treeRoot, "XAML Failed to Load");
+
+                // Initialize Visual Tree
+                await SetTestContentAsync(treeRoot);
+
+                // Main Test
+                var textBlock = treeRoot.FindDescendant<TextBlock>(searchType);
+
+                Assert.IsNotNull(textBlock, "Expected to find something.");
+                Assert.IsInstanceOfType(textBlock, typeof(TextBlock), "Didn't find expected typed element.");
```

Comment:
First parameter should be expected value

Reply:
Should it though 😄

![image](https://user-images.githubusercontent.com/10199417/146769343-62d95a15-c5ad-4bf4-8182-697337d7dfae.png)